src/Core/src/Platform/Windows/SwitchExtensions.cs

```diff
@@ -0,0 +1,12 @@
+﻿using Microsoft.UI.Xaml.Controls;
+
+namespace Microsoft.Maui
+{
+	public static class SwitchExtensions
+	{
+		public static void UpdateIsToggled(this ToggleSwitch toggleSwitch, ISwitch view)
+		{
+			toggleSwitch.IsOn = view.IsToggled;
```

Comment:
I don't know how this got in, but the Core property should not be `ISwitch.IsToggled` - it should be `ISwitch.IsOn`.